.github/styles/Vocab/Products/accept.txt

```diff
@@ -1,44 +1,81 @@
+721s
 ABIs
 addValidator
 AddValidator
+Agrawal
+aggresive
+Aggresive
```

Comment:
these are wrong spelling, right? it should 2 s in aggressive.

Reply:
Confirmed